lib/Doctrine/Migrations/FileQueryWriter.php

```diff
@@ -41,7 +41,7 @@ public function write(
         array $queriesByVersion,
         ?DateTimeInterface $now = null
     ) : bool {
-        $now = $now ?? new DateTime();
+        $now = $now ?? new DateTimeImmutable();
```

Comment:
Does it make sense to use `new DateTimeImmutable('now', new DateTimeZone('UTC'));` like above in Configuration.php? Can a timezone with daylight saving time an issue here?

Reply:
Probably should be changed to match elsewhere. Practically I don't think it would ever cause an issue though.

Reply:
I think it needs good timing. But this is for another time and PR to find out.

Reply:
I will send another PR to fix that.

Reply:
Issue logged here https://github.com/doctrine/migrations/issues/776